more-dom-manipulation/app.js

```diff
@@ -0,0 +1,83 @@
+//1. Change the font:
+const pageFont = document.querySelector(".page");
+pageFont.style.fontFamily = "Arial, sans-serif";
+
+//2.Replace each of the span tags
+//NickName:
+const sTags = document.querySelector("span#nickname");
+sTags.replaceWith("Eman");
```

Comment:
in order to put your nickname in the nickname span without removing the span itself use `innetText` instead of `replaceWith`